packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -139,11 +157,35 @@ class AccountsBloc extends Bloc implements AccountsBlocEvents, AccountsBlocState
     setActiveAccount(account);
   }
 
+  /// The currently active account.
+  ///
+  /// Equivalent to activeAccount.value but throws a [StateError] when not user is logged in.
+  @visibleForTesting
+  Account get aa {
```

Comment:
Could this just be private?

Reply:
It basically is private.

Also as every dart file we have is `part of` neon there isn't anything private in our code anyways.

Also we want to unit test it eventually so why not from the beginning?

Reply:
But the method could be private to the class, right? I agree for testing it is useful but then it should have a better name.

Reply:
In dart there isn't any concept of private to the class.
In dart things with an underscore are private to it's package (aka it's file). So visibleForTesting == leadingUnderscore (only difference is that one can be used in tests).

If you really want the 'private' aspect we should rethink having all files `part`ed together.

Reply:
Oh you are teaching me new dart things everyday :smile:
I always thought that there are private methods for classes...

Reply:
Yeah I just tried it myself already